apps/src/code-studio/header.js

```diff
@@ -172,6 +173,8 @@ function shareProject() {
     const appType = dashboard.project.getStandaloneApp();
     const pageConstants = getStore().getState().pageConstants;
     const canShareSocial = !pageConstants.isSignedIn || pageConstants.is13Plus;
+    const canPublish = !!appOptions.isSignedIn &&
+      PUBLISHED_PROJECT_TYPES.includes(appType);
```

Comment:
You had me convinced for a moment, but then I decided to try it out in the Chrome debugger just for fun:
<img width="147" alt="screen shot 2017-08-09 at 5 08 33 pm" src="https://user-images.githubusercontent.com/8001765/29148951-79a2364a-7d25-11e7-945e-e1600b048aa1.png">
as you can see, if the first value is falsy, it gets returned without being coerced into a boolean.